### PR TITLE
Notebook fix to handle GCS outputs and resolve (AutoML Tabular Forecasting notebook error: no row field 'name' #453)

### PR DESCRIPTION
**Pull request by manuelamunategui**
This PR contains a notebook that demonstrates how to use the Vertex SDK to create tabular forecasting models and do batch prediction using a Google Cloud [AutoML](https://cloud.google.com/vertex-ai/docs/start/automl-users) model.

- [ x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x ] Follow the style and grammar rules outlined in the above notebook template.
- [x ] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x ] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).
- [ x] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under `# Official Notebooks` section, pointing to the author or the author's team.
- [ x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/automl/sdk_automl_tabular_forecasting_batch.ipynb [hunk 1]
```diff
@@ -8,7 +8,7 @@
       },
```
Comment (andrewferlitsch) [inline notebooks/official/automl/sdk_automl_tabular_forecasting_batch.ipynb:8]: <p>missing the f"...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/477/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

File: notebooks/official/automl/sdk_automl_tabular_forecasting_batch.ipynb [hunk 2]
```diff
@@ -814,7 +804,8 @@
         "    job_display_name=f\"iowa_liquor_sales_forecasting_predictions_{TIMESTAMP}\",\n",
         "    bigquery_source=PREDICTION_DATASET_BQ_PATH,\n",
         "    instances_format=\"bigquery\",\n",
-        "    bigquery_destination_prefix=batch_predict_bq_output_uri_prefix,\n",
+        "    # bigquery_destination_prefix=batch_predict_bq_output_uri_prefix,\n",
```
Comment (ivanmkc) [inline notebooks/official/automl/sdk_automl_tabular_forecasting_batch.ipynb:807]: Seems that this line needs to be reverted to maintain original purpose of showing a bq destination.

**Review by ivanmkc — CHANGES_REQUESTED** [review]

File: notebooks/official/automl/sdk_automl_tabular_forecasting_batch.ipynb (hunk 2 above)
Comment (manuelamunategui) [reply notebooks/official/automl/sdk_automl_tabular_forecasting_batch.ipynb:816]: The batch_predict can do either or, not both - as the reset of the rest of the notebook uses GCS, I opted with it (I added some markup to explain the difference)

**Review by andrewferlitsch — APPROVED** [review]

**Comment by manuelamunategui** [comment]
Notebook works in both workbench and colab - confirmed

**Pull request merged**